### ISSUE TITLE
web2's docstring should indicate the stability of web2

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2028]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#2028|
|Type|enhancement|
|Created|2006-08-29 07:07:40Z|

Ideally, the docstring will explain how stable it is, why it is not ready to replace t.web and also why it's not just a web server.

<details><summary>Searchable metadata</summary>

```
trac-id__2028 2028
type__enhancement enhancement
reporter__jml jml
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web2 web2
keywords__documentation documentation
time__1156835260000000 1156835260000000
changetime__1213795969000000 1213795969000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun removed owner|
|-|-|

```
Index: twisted/web2/__init__.py
===================================================================
--- twisted/web2/__init__.py    (revision 23937)
+++ twisted/web2/__init__.py    (working copy)
@@ -1,12 +1,15 @@
 # -*- test-case-name: twisted.web2.test -*-
-# Copyright (c) 2001-2004 Twisted Matrix Laboratories.
+# Copyright (c) 2001-2008 Twisted Matrix Laboratories.
 # See LICENSE for details.
 
 
 """
+Twisted Web2: an experimental web server implementation.
 
-Twisted Web2: a better Twisted Web Server.
+Feature from this package are being merged into L{twisted.web}.  Once that
+is complete, this package will be removed.
 
+See U{http://twistedmatrix.com/trac/wiki/WebDevelopmentWithTwisted}.
 """
 
 from twisted.web2._version import version
```

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_2"></a>|@jml set owner to @exarkun|
|-|-|

Looks great. Please merge it.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [23951]) Apply doc patch clarifying the state of twisted.web2

Author: exarkun
Reviewer: jml
Fixes: [#2028](https://github.com/twisted/twisted/issues/2028)

Explain what web2 is, what is being done with it, and link to the wiki page
for more details.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
